### DISCUSSION TITLE
Handle Creation or Update Data on Simple Resource

### DISCUSSION THREAD

**Discussion by lutfifirdaus** [filamentphp/filament#4403] [Help] [answered]
Hello everyone.

Is anybody know how to handle create or update data before store on simple/modal resource? Or is handle create or update data only possibly can done  via `handleRecord` function on `CreateRecord | UpdateRecord` page?

**Comment by danharrin** [comment, marked as answer]
https://filamentphp.com/docs/2.x/admin/resources/listing-records#customizing-data-before-saving

**Reply by lutfifirdaus** [reply]
Thank you man
